packages/fs-detectors/src/monorepos/get-monorepo-default-settings.ts

```diff
@@ -74,17 +83,25 @@ export async function getMonorepoDefaultSettings(
     if (projectPath === '/') {
       return {
         monorepoManager: 'turbo',
-        buildCommand: 'npx turbo run build',
+        buildCommand: 'turbo run build',
         installCommand: packageManager ? `${packageManager} install` : null,
         commandForIgnoringBuildStep: 'npx turbo-ignore',
       };
     }
 
+    let buildCommand = null;
+    if (projectPath) {
+      if (turboSemVer && !semver.intersects(turboSemVer, '<1.8.0')) {
```

Comment:
i've found `semver.satisfies` to be a bit clearer 

```suggestion
      if (turboSemVer && semver.satisfies(turboSemVer, '<1.8.0')) {
```

Reply:
We can't switch to satisfies. We need to check to see if the version could *possibly* match anything less than 1.8.0 since we're not querying the lockfile. This is intentionally an intersection check.